pallets/liquid-staking/src/lib.rs

```diff
@@ -483,6 +501,7 @@ pub mod pallet {
             let new_unstake = UnstakeInfo {
                 amount,
                 block_number,
+                era_index: Some(era_index),
```

Comment:
Era index should be updated after the offchain staking client process it successfully.

Reply:
As I know, the Era index should be fixed after we unbond on relaychain. if we unbond a second time on relaychain, it is totally another Era index.
The era index is associated with the amount each time invoking `unbond`.
please correct me if I have any misunderstandings.